src/billing/components/PayAsYouGo/CancelServiceContext.tsx

```diff
@@ -24,6 +24,12 @@ const RedirectLocations = {
 }
 const DEFAULT_REDIRECT_LOCATION = '/mkt_cancel'
 
+const getDefaultCancelationReason = () => {
+  return Object.keys(CancelationReasons).filter(
+    reason => CancelationReasons[reason] === CancelationReasons.NONE
+  )[0]
+}
+
```

Comment:
This (which is a little gross) is to resolve the issue with line 48. Enums are being used to track the user's selected 'cancelation reason', and 'reason' represents the current _key_ of the enum reflecting that cancelation reason. Having that remain a string like 'NO_OPTION' is a big no-no imho, because someone changing the enum won't know to search for that string.

The above ^ is marginally better, because even though the logic of what it's doing is stupid (find me the name of the key that corresponds to `CancelationReasons.NONE`, it isn't expensive in a way we care about, and at least means that changes to the enum names will work correctly without having to run more manual searches.